variants/DISCO_F100RB/variant.h

```diff
@@ -133,7 +133,7 @@ enum {
 
 //SPI definitions
 //define 16 channels. As many channel as digital IOs
-#define SPI_CHANNELS_NUM        16
+#define SPI_CHANNELS_NUM        PEND
```

Comment:
Pleaser use NUM_DIGITAL_PINS instead of PEND for all
Amend the commit to avoid twice commit for this

Reply:
Done!

Reply:
Thanks! ;)